scripts/admincommands.js

```diff
@@ -147,22 +147,22 @@ exports.handleCommand = function(src, command, commandData, tar, channel) {
         var regex;
         try {
             regex = new RegExp(commandData.toLowerCase()); // incase sensitive
-        } catch (e) {

```

Comment:
probably something else causing that, such as a lack of if-filtering or because lint is dumb sometimes and you can't trust machines for everything